static/app/utils/metrics/useMetricsCardinality.tsx

```diff
@@ -0,0 +1,52 @@
+import type {MRI} from 'sentry/types';
+import type {Project} from 'sentry/types/project';
+import {useMetricsQuery} from 'sentry/utils/metrics/useMetricsQuery';
+
+type Props = {
+  project: Project;
+};
+
+const CARDINALITY_QUERIES = [
+  {
+    name: 'a',
+    mri: 'g:metric_stats/cardinality@none' as MRI,
+    op: 'max',
+    groupBy: ['mri'],
+    query: '!mri:"" cardinality.window:3600',
+    orderBy: 'desc' as 'desc' | 'asc',
+  },
+];
+
+const CARDINALITY_DATE_TIME = {
+  period: '1h',
+  start: null,
+  end: null,
+  utc: null,
+};
+
+const CARDINALITY_INTERVAL = '1h';
```

Comment:
You mentioned 24 hours previously. Are we looking only at the last hour in the end?